analytics-integrations/moengage/src/main/res/layout/moe_image_message_button.xml

```diff
@@ -0,0 +1,63 @@
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
What are these layout files for? Seems like these should be package in the MoEngage library.